doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+
+Given a classifier $f$, a value $\tau>0$, an instance $x$, and a data distribution $\mathcal{D}$, $A$ is an anchor for $x$ if $A(x) = 1$ and,
+
+$$
+E_{\mathcal{D}(z|A)}[1_{f(x)=f(z)}] \geq \tau
+$$
+
+The distribution $\mathcal{D}(z|A)$ is those points from the dataset for which the anchor holds. This is like fixing some set of features of an instance and allowing all the others to vary. This condition says any point in the data distribution that satisfies the anchor $A$ is expected to match the model prediction $f(x)$ with probability $\tau$ (usually $\tau$ is chosen to be 0.95).
+
+Let $prec(A) = E_{\mathcal{D}(z|A)}[1_{f(x)=f(z)}]$ be the precision of an anchor. Note that the precision of an anchor is considered with respect to the set of points in the data distribution to which the anchor applies, $\mathcal{D}(z|A)$. We can consider the **coverage** of an anchor as the probability that $A(z)=1$ for any instance $z$ in the data distribution. The coverage tells us the proportion of the distribution to which the anchor applies. The aim here is to find the anchor that applies to the most extensive set of instances while satisfying $prec(A) \geq \tau$.
+
+__TODO__:
+- Include picture explanation of anchors
+
+Alibi finds anchors by building them up from the bottom up. We start with an empty anchor $A=\{\}$ and then consider the set of possible feature values from the instance of interest we can add. $\mathcal{A} = \{A \wedge a_0, A \wedge a_1, A \wedge a_2, ..., A \wedge a_n\}$. For instance, in the case of textual data, the $a_i$ might be words from the instance sentence. In the case of image data, we partition the image into super-pixels which we choose to be the $a_i$. At each stage, we will look at the set of possible next anchors that can be made by adding in a feature $a_i$ from the instance. We then compute the precision of each of the resulting anchors and choose the best. We iteratively build the anchor up like this until the required precision is met.
+
+As we construct the new set of anchors from the last, we need to compute the precisions of the next group to know which one to choose. We can't calculate this directly; instead, we sample from $\mathcal{D}(z|A)$ to obtain an estimate. To do this, we use several different methods for different data types. For instance, in the case of textual data, we want to generate sentences with the anchor words and ensure that they make sense within the data distribution. One option is to replace the missing words (those not in the anchor) from the instance with words with the same POS tag. This means the resulting sample will make semantic sense rather than being a random set of words. Other methods are available. In the case of images, we sample an image from the data set and then superimpose the super-pixels on top of it.
```

Comment:
These also likely belong in the Anchor-specific docs. Although some (informal) discussion of how the algorithm runs would be good to have - mostly to convey the point that it's a sequential procedure which implies complexity concerns. Could keep the same text example.

Reply:
yes, on the sampling a brief explanation for tabular, images and text (2 types) would be fine (just a few lines on each) with reference to the Anchor method page but highlight in a "pitfall" note that this sampling is key to the results.